cmd/nse/main.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
Why are we building cmd/ s into the integration tests rather than using cmd-* repos?